README.md

```diff
@@ -17,3 +18,17 @@ Please contact your technical support representative for any questions about the
 
 ## License
 [Apache License v2.0](LICENSE)
+
+## Contributing
+
+Any contribution are welcome, as an issue or pull request. 
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
```

Comment:
via issue -> by creating an issue.
I would not require creating an issue for small changes such as: typos, obvious programming mistakes. However, creating an issue is recommended for: API changes, feature changes, bug fixes, improvements etc...

Reply:
Reworded